### PR TITLE
Feature/111 manual dualstack docs

### PR DESCRIPTION
**Pull request by adamhgriffith-uofu**
Re-wrote the manual SLATE cluster installation documentation pages using the steps discovered while getting [adamhgriffith-uofu/
dualstack-virtualbox-k8s](https://github.com/adamhgriffith-uofu/dualstack-virtualbox-k8s) up and running.

* Passing over to @LincolnBryant and @muali14  for review as per @amblinalong.
* Please squash-merge this pull request as the branch history is not useful.

### CODE REVIEW

**Comment by LincolnBryant** [comment]
Any idea why the selenium tests are failing ?

File: _docs/cluster/manual/slate-worker-node.md [hunk 1]
```diff
@@ -10,16 +10,98 @@ type: markdown
...
+   # master node
+   MASTER_IPV4_ADDR=192.168.0.3
+   MASTER_IPV6_ADDR=2001:db8:1234:5678::1
+   # worker node
+   WORKER_IPV4_ADDR=192.168.0.4
+   WORKER_IPV6_ADDR=2001:db8:1234:5678::2
+   ```
+   {:data-add-copy-button='true'}
+
+## Build Kubeadm Join
```
Comment (LincolnBryant) [inline _docs/cluster/manual/slate-worker-node.md:53]: Why is this needed? Strictly for IPv6?

**Review by LincolnBryant — APPROVED** [review]
I think if you can fix up the selenium results then we should be good to merge. This looks like a good set of changes.

**Comment by adamhgriffith-uofu** [comment]
> Any idea why the selenium tests are failing ?

The Selenium tests are failing due to #110 .

**Comment by adamhgriffith-uofu** [comment]
@muali14 is experiencing syntax errors for the `kubadm init` YAML. I will look into that this afternoon.

**Comment by adamhgriffith-uofu** [comment]
Fixed first set of errors @muali14 was experiencing in 2321c66 (IPv6 address :: needed to be in quotes).

File: _docs/cluster/manual/slate-worker-node.md (hunk 1 above)
Comment (adamhgriffith-uofu) [reply _docs/cluster/manual/slate-worker-node.md:62]: I have added comments to the following to clarify in `slate-worker-node.md`:
```shell
 # master node
   MASTER_IPV4_ADDR=203.0.113.3               # required if the control-plane is IPv4
   MASTER_IPV6_ADDR=2001:db8:1234:5678::1     # required if the control-plane is IPv6
```
These variables are used further down the doc page in either the IPv4 or IPv6 join YAML templates. Does that answer your question @LincolnBryant ?
